src/main/java/frc/robot/commands/auto/AutoRoutines.java

```diff
@@ -97,26 +91,20 @@ public Command getAutonomousCommand(int selectAuto) {
             case 1:
                 return centerMove;
             case 2:
-                // robot start on right corner of starting line (robot perspective) or left from driver perspective
-                return oneL4Coral_rightBlue;
-            case 3:
-
-                // robot start on left corner of starting line (robot perspective) or right from driver perspective 
                 return oneL4Coral_leftBlue;
+            case 3:
+                return oneL4Coral_rightBlue;
             case 4:
-                return RIGHTCenter_R_CD_R;
+                return testVisionAuto;
             case 5:
-                return LEFTCenter_R_CD;
+                return NOVISION_oneL4Coral_leftBlue;
             case 6:
-                return LEFTCenter_R_CD_R;
+                return threeL4Coral;
             case 7:
-                return Right_R_CD_R;
+                return oneL4Coral_CD;
```

Comment:
If you plan to use id > 5, please confirm if it is accurate with analog selector.